packages/gafl-webapp-service/src/processors/__tests__/date-and-time-display.spec.js

```diff
@@ -34,6 +43,39 @@ describe('displayStartTime', () => {
     expect(moment.utc).toHaveBeenCalledWith(startDate, null, expect.any(String))
   })
 
+  it.each([[50], [10000], [0]])(
+    'expected text replaces the date string if start after payment is selected using correct constant',
```

Comment:
It's good to include a parameter within the test description so that you can see the difference between the tests in the output. You can include it by putting %s into the test description, for example 'generates a string of payment delay (%s) followed by licence_summary_minutes_after_payment label when licence is to start after payment'

Also, you're varying the `START_AFTER_PAYMENT_MINUTES` constant, you should put `startDate` and the `licence_summary_minutes_after_payment` message in there too and vary them a bit